### PR TITLE
Upgrade the CLI version

### PR DESCRIPTION
**Pull request by victorhurdugaci**
cc @pranavkm @ryanbrandenburg @rynowak

### CODE REVIEW

File: build/cli.version.win [hunk 1]
```diff
@@ -1 +1 @@
-1.0.0.001540
\ No newline at end of file
+1.0.0.001651
```
Comment (pranavkm) [inline build/cli.version.win]: Why is this different?

Comment (victorhurdugaci) [inline build/cli.version.win]: Because latest windows != latest other today

**Comment by pranavkm** [comment]
:shipit:

**Comment by rynowak** [comment]
Can we hold off on this for like 30 mins? A fix we need is building right now

**Comment by NTaylorMullen** [comment]
Once this is in we should re-transition all of our tools back over to netstandardapp1.5. All should be good supposedly.

**Comment by pranavkm** [comment]
> All she be good supposedly.

Wut?

**Comment by victorhurdugaci** [comment]
> Can we hold off on this for like 30 mins? A fix we need is building right now

Does it require a new CLI?

**Comment by NTaylorMullen** [comment]
> Wut?

It'll also contain fixes for restoring CLI tools that are netstandardapp1.5. We initially transitioned all of our tools to netstandardapp1.5 and then reverted the transition after the CLI was found to not support it.

**Comment by pranavkm** [comment]
:clap:

**Pull request merged**